packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -137,13 +137,21 @@ export async function sendSignedXmlMtom({
         },
         httpsAgent: agent,
         responseType: "arraybuffer",
+        maxBodyLength: Infinity,
+        maxContentLength: Infinity,
```

Comment:
https://github.com/axios/axios/issues/1362
^ we have had this error in prod 3 times

`maxContentLength size of -1 exceeded`

We added retries for ERR_BAD_RESPONSE, and this change was also recommended